ketcher-autotests/tests/Macromolecule-editor/Erase-Tool/erase-tool.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { test } from '@playwright/test';
+import { selectEraseTool, selectSingleBondTool } from '@utils';
+import { turnOnMacromoleculesEditor } from '@utils/macromolecules';
+import { bondTwoMonomers } from '@utils/macromolecules/polymerBond';
+/* eslint-disable no-magic-numbers */
+
+test.describe('Erase Tool', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('');
+  });
+  test.skip('Delete monomer bonded with another monomers', async ({ page }) => {
+    /* 
+    Test case: #2370 - "Erase" tool for macromolecules editor
+    Description: Erase Tool
+    */
+
+    await turnOnMacromoleculesEditor(page);
+
+    // Choose peptide
+    await page.getByText('Tza').click();
+
+    // Create 4 peptides on canvas
+    await page.mouse.click(300, 300);
+    await page.mouse.click(400, 400);
+    await page.mouse.click(500, 500);
+    await page.mouse.click(500, 200);
+
+    // Get 4 peptides locators
+    const peptides = await page.getByText('Tza').locator('..');
+    const peptide1 = peptides.nth(0);
+    const peptide2 = peptides.nth(1);
+    const peptide3 = peptides.nth(2);
+    const peptide4 = peptides.nth(3);
+
+    // Select bond tool
+    await selectSingleBondTool(page);
+
+    // Create bonds between peptides
+    await bondTwoMonomers(page, peptide1, peptide2);
+    await bondTwoMonomers(page, peptide3, peptide2);
+    await bondTwoMonomers(page, peptide3, peptide4);
+
+    await page.screenshot({
+      path: 'tests/Macromolecule-editor/screenshots/erase-tool.png',
```

Comment:
I propose to use a function `takeEditorScreenshot`. It automatically creates screenshot of canvas area and generates the screenshot name